patching/psp.md

```diff
@@ -0,0 +1,59 @@
+# Using Refresher to setup Allefresher on PSP/Vita
+
+Refresher has support for automatically deploying Allefresher onto a PSP or Vita, with the relavent configuration automatically filled out. This will let you connect to a custom server on LittleBigPlanet for the PSP (on Vita, this would be through Adrenaline).
+
+It also assumes that you've already downloaded Refresher. If you haven't, head on over to the [GitHub page](https://github.com/LittleBigRefresh/Refresher) to grab yourself a copy.
+
+![A Screenshot of Refresher's PSP patching screen](https://littlebigrefresh.github.io/Docs/patching/images/refresher-psp.png)
+
+## Step 1: Jailbreak your device and install a CFW
+
+We can't help you with this, but there are various guides out there you can go out and find. We personally recommend the guides on [pspunk](https://www.pspunk.com/psp-cfw/) and [vita.hacks.guide](https://vita.hacks.guide/), for PSP and Vita respectively.
+
+### PSP
+
+Try to use the latest firmware, and we reccomend PRO C over other CFWs, as they have not been thoroughly tested with Allefresher yet. It is possible to dual-boot CFWs, so if you need a CFW such as ARK-4 for other homebrew, we'd reccomend dual-booting PRO C as well until Allefresher is guarenteed to work well on ARK-4.
+
+### Vita
+
+You'll need to setup Adrenaline, and use the latest version of the Adrenaline ePSP firmware (as with PSP, ARK-4 is untested).
+
+## Step 2: Connect your PSP/Vita or Memory Stick to your computer
```

Comment:
Doesn't Vita require special steps to get filesystem access over USB?

Reply:
> Doesn't Vita require special steps to get filesystem access over USB?

Not sure, im writing the vita instructions assuming the person reading it knows how to get USB to work in vitashell, as i dont know the exact steps